kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/pom.xml

```diff
@@ -442,6 +457,16 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.xmlunit</groupId>
+      <artifactId>xmlunit-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.xmlunit</groupId>
+      <artifactId>xmlunit-assertj</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Could you doublecheck please these are not in droolsjbpm-build-bootstrap?

Reply:
Will do.. this is PoC (as these would need moving to `droolsjbpm-build-bootstrap` too).

Reply:
Yes `org.xmlunit:xmlunit-core` is in `kie-parent` but version `2.3.0`. I'll submit a PR to you for `droolsjbpm-build-bootstrap` pushing to 2.6.3 (hopefully it won't break things). `org.xmlunit:xmlunit-assertj` is not declared yet.

Reply:
I have a PR ready to submit to `jomarko/droolsjbpm-build-bootstrap` but it's not listed in GitHub.

IDK if you have to grant me access? I could create this one for `jomarko/kie-wb-common` OK.

https://github.com/manstis/droolsjbpm-build-bootstrap/tree/KOGITO-778

Reply:
not sure, created it here https://github.com/jomarko/droolsjbpm-build-bootstrap/pull/1/files